Framework/Kernel/inc/MantidKernel/DataService.h

```diff
@@ -482,15 +482,26 @@ template <typename T> class DLLExport DataService {
   }
 
   /// Get a vector of the pointers to the data objects stored by the service
-  std::vector<boost::shared_ptr<T>> getObjects() const {
+  std::vector<boost::shared_ptr<T>>
+  getObjects(DataServiceHidden includeHidden = DataServiceHidden::Auto) const {
     std::lock_guard<std::recursive_mutex> _lock(m_mutex);
 
-    const bool showingHidden = showingHiddenObjects();
+    if (includeHidden == DataServiceHidden::Include) {
+      // Return all elements including hidden ones
```

Comment:
Does this if block do anything currently ?

Reply:
No, good catch - it's a leftover from before refactoring